lib/ex_aws/s3.ex

```diff
@@ -848,14 +848,31 @@ defmodule ExAws.S3 do
     request(:put, bucket, object, headers: headers, resource: "acl")
   end
 
-  @doc "Add a set of tags to an existing object"
+  @doc """
+  Add a set of tags to an existing object
+
+  ## Options
+
+  - `:version_id` - The versionId of the object that the tag-set will be added to.
+
+  """
   @spec put_object_tagging(
           bucket :: binary,
           object :: binary,
           tags :: Access.t(),
-          opts :: Keyword.t()
+          opts :: [option]
         ) :: ExAws.Operation.S3.t()
+        when option: {:version_id, binary} | {atom, binary}
```

Comment:
I'm not quite clear why this change has been made. The type `{atom, binary}` should already cover `{:version_id, binary}`, and indeed both types are already covered by `Keyword.t()` as was in the original version.

Reply:
@bernardd You're right 👍  I had missed it. Fixed in 478aa61.

Reply:
Thanks - sorry to be a pain, but can you wind the typespec back to how it was? That is, `opts` just defined as a `Keyword.t()` rather than its own type? Then we're good to merge - thanks :)

Reply:
@bernardd Okay, reverted in 575465f